src/core/search/compressed_sorted_set.h

```diff
@@ -50,6 +51,8 @@ class CompressedSortedSet {
   friend struct Iterator;
 
  public:
+  explicit CompressedSortedSet(std::pmr::memory_resource* mr);
```

Comment:
it does not work well for some older platforms. Please see how I do it in bptree_set.h
specifically you need to include `#include "base/pmr/memory_resource.h"` and not `<memory_resource>` and use `PMR_NS`